### DISCUSSION TITLE
In GH Project I want to group by multiple columns to created a nested view

### DISCUSSION THREAD

**Discussion by mtcA6** [community/community#60154] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

In my project I've grouped by sprints/iterations 
![image](https://github.com/community/community/assets/129431463/227bf836-e8df-44b3-a1e8-f130c3486e9d)
now i'd like to be able to group by additional fields, eg assignee. 

- Sprint 
     - assignee/practice area 
In this case we want to see how many issues are assigned to a person per sprint without having to filter and unfilter the board
     
- Sprint 
     - tracked by 
in this example i want to see how many pieces of work per 'epic'/'initiative' we're bringing into a sprint. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by roconno1** [comment]
Just weighing in here to say that I too would appreciate this feature.
